src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -137,4 +137,10 @@ class ReportController(
         reportRepository.publish(ids)
         return okayResponse()
     }
+
+    fun getRunningReportsDetails(): ReportRunLog
```

Comment:
This should be in `ReportRunController` with all the other endpoints that expect a `key`

Reply:
..and maybe should be called getRunningReportLogs to match endpoint and class